api/src/main/resources/liquibase.xml

```diff
@@ -731,4 +731,15 @@
         <addForeignKeyConstraint constraintName="patient_appointment_fulfilling_encounter_map_patient_appointment" baseTableName="patient_appointment_fulfilling_encounter_map" baseColumnNames="patient_appointment_id" referencedTableName="patient_appointment" referencedColumnNames="patient_appointment_id"/>
         <addForeignKeyConstraint constraintName="patient_appointment_fulfilling_encounter_map_encounter" baseTableName="patient_appointment_fulfilling_encounter_map" baseColumnNames="fulfilling_encounter_id" referencedTableName="encounter" referencedColumnNames="encounter_id"/>
     </changeSet>
+    <changeSet id="add_date_appointment_scheduled_20240528" author="aojwang">
+        <preConditions onFail="MARK_RAN">
+            <not>
+                <columnExists columnName="date_appointment_scheduled" tableName="patient_appointment"/>
+            </not>
+        </preConditions>
+        <sql>
```

Comment:
Instead of SQL, preferably use <addColumn>
```

<addColumn tableName="patient_appointment">
            <column name="date_appointment_scheduled" type="DATE"/>
</addColumn>

Because in future even a formatting will cause the changeset hash to be different. 
```

Reply:
Thanks @angshu

Reply:
I have updated the code. Please re-review. @angshu @mogoodrich @mseaton

Reply:
Do you want the datatype to be just DATE or DATETIME? 
Also, I think you need a Datetime column, If you want to copy the data from date_created and populate this new column

Reply:
DATETIME works fine as well. I have made the changes. Please re-review